### ISSUE TITLE
[bug] foreach usage possible: incomplete quick fix

### ISSUE THREAD

**Issue by orklah** [kalessil/phpinspectionsea#1186]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug                      |
| Plugin         | Php Inspections (EA Extended) |

Current behaviour (description/screenshot:)
--
Apply the "foreach usage possible" quick fix on the for loops in the following code:
```php
<?php
function foo()
{
    $array = [];
    $nb = count($array);

    for ($i = 0; $i < $nb; $i++) {
        echo " $array[$i]";
    }

    for ($i = 0; $i < $nb; $i++) {
        echo $array[$i]['random_index'];
    }

    for ($i = 0; $i < $nb; $i++) {
        echo "{$array[$i]}";
    }
}
``` 
On every case, the $array[$i] isn't modifier by the quickfix

Expected behaviour
--
$array[$i] should be replaced by $iValue by the quick fix.

Thanks!

**Comment by kalessil** [comment]
Fixed except inlined strings, will check later if inlined case can be fixed as well.

**Comment by kalessil** [comment]
Also done for the inlined cases.

**Comment by orklah** [comment]
Thanks! Glad to see you're back :)

**Comment by kalessil** [comment]
Yep, offline topics took a few weeks =)
